int32 workerID
---
bool isActive
bool isComplete
int32 workerID
string type
int32 zone
Pose startingPoint
